README.md

```diff
@@ -1,13 +1,24 @@
-## My Project
+## Ion Hash Test Driver
+A cross-implementation test driver for [Amazon Ion Hash](https://amzn.github.io/ion-hash/) readers and writers.
 
-TODO: Fill this README out!
+[![Build Status](https://travis-ci.org/amzn/ion-hash-test-driver.svg?branch=master)](https://travis-ci.org/amzn/ion-hash-test-driver)
 
-Be sure to:
+## Usage
+The Ion Hash Test Driver fetches copies of known Ion Hash implementations and executes each against a set of test data derived from
+[ion-hash-test](https://github.com/amzn/ion-hash-test).  It then generates a report indicating how consistent the implementations
+are when hashing the test data.
 
-* Change the title in this README
-* Edit your repository description on GitHub
+To execute the test driver:
+```
+$ python3 -m venv venv
+$ . venv/bin/activate
+$ pip install -r requirements.txt
+$ pip install -e .
+$ python ionhashtest/ion_hash_test_driver.py
+```
 
-## License
+Additional Ion Hash implementations may be tested by adding a command-line tool such as [command-line tool](https://github.com/amzn/ion-hash-python/blob/master/tools/ion-hash), then adding information about the implementation to this project's [config.py](https://github.com/amzn/ion-hash-test-driver/blob/master/ionhashtest/config.py) file.
```

Comment:
This link 404s for me: https://github.com/amzn/ion-hash-python/blob/master/tools/ion-hash

I think it would be worth defining here what the CLI must look like so the definition is in a centralized location.

Reply:
That link succeeds here, now that tools/ion-hash has been merged into ion-hash-python/master.

I've captured some detail about the expected tool interface and behavior.